### DISCUSSION TITLE
LDAP authentication is loading forever

### DISCUSSION THREAD

**Discussion by reisadriel** [nilsteampassnet/TeamPass#4410] [Q&A]
Hello everyone, I need help to solve this issue and I believe the problem is in the topic "Base DN", I don't know how to organize it right now, the structure that I'm using is "cn=users,dn=domain admins,dc=my,dc=domain;" it's in the 3.1.1 version, I would apprecciate if someone explain how each one of the topics work, since I didn't find much information about it in the internet.

**Comment by Hug0-stiglitz** [comment]
Hello! Did you solve this problem?
